src/AltinnCore/Authentication/JwtCookie/JwtCookieDefaults.cs

```diff
@@ -17,24 +15,6 @@ public static class JwtCookieDefaults
         /// </summary>
         public static readonly string CookiePrefix = ".AspNetCore.";
```

Comment:
this was the cause of one of our bugs? Could this also be removed or replaced with an empty string?